src/en/send.md

```diff
@@ -353,3 +341,145 @@ If the request is not successful, the response body is `json`, refer to the tabl
+{
+  "name": "My bulk name",
+  "template_id": "f33517ff-2a88-4f6e-b855-c550268ce08a",
+  "rows": [
+    ["email address", "name"],
+    ["alice@example.com", "Alice"],
+    ["bob@example.com", "Bob"]
+  ],
+  "scheduled_for": "2021-06-08T15:15:00", # optional string
+  "sender_id": "f025b1a9-63af-43e8-b969-627bfe544bba" # optional string
+}
+```
+
+### Arguments
+
+**name (required)**
+
+The `name` of your bulk sending. Used to identify this batch of notifications later on.
+
+<Content :page-key="$site.pages.find(p => p.relativePath === 'en/_arg_template_id.md').key"/>
+
+**rows (required)**
+
+An array of arrays. The first line is the header and should include at least `email address` if you're sending an email template or `phone number` if you're sending a text message template. The other column headers should match placeholder fields of your template.
+
+The following lines should be your recipients' details and should match the order of column headers. You can have between 1 and 50,000 recipients.
+
+#### Optional arguments
+
+**scheduled_for (optional)**
+
+If you want to send your batch of notifications in the future, you can specify a datetime up to 4 days in the future, in [ISO 8601 format](https://en.wikipedia.org/wiki/ISO_8601). Example: `2021-06-08T15:15:00` (UTC time).
+
+**sender_id (optional)**
+
+If you want to use a specific reply-to email address when sending a batch of emails, you can specify a reply-to email address ID.
+
+To find your reply-to email address ID:
+
+1. [Sign in to GC Notify](https://notification.canada.ca/sign-in).
+1. Go to the __Settings__ page.
+1. In the __Email__ section, select __Reply-to email addresses__
+1. Copy the ID of the reply-to email address you want to use
+
+By default, GC Notify will use your default reply-to email address if you don't specify one, or none if you didn't set up one.
+
+**csv (optional)**
+
+If you prefer to pass the content of CSV files instead of rows in the `rows` argument, you can do so. Pass the full content of your CSV file in a key named `csv`. Do not include the `rows` argument.
+
+For example
+
+```json
+{
+  "name": "My bulk name",
+  "template_id": "f33517ff-2a88-4f6e-b855-c550268ce08a",
+  "csv": "email address,name\nalice@example.com,Alice"
+}
+```
+
+
+### Response
+
+::: warning Response timeout
+If you specify a response timeout when calling this endpoint, make sure it is set to 15 seconds. The GC Notify API could take a few seconds to validate your request and save the payload if you submit a large batch.
+:::
+
+If the request is successful, the response body is `json` with a status code of `201`:
+
+```json
+{
+   "data":{
+      "api_key":{
+         "id":"de1fafa2-fb2a-49c5-9b9a-8400727ecd29",
+         "key_type":"team",
+         "name":"Test"
+      },
+      "archived":false,
+      "created_at":"2021-06-10T17:14:15.341308+00:00",
+      "created_by":{
+         "id":"6af522d0-2915-4e52-83a3-3690455a5fe6",
+         "name":"Notify service user"
```

Comment:
- What is "Test" referring to here the name of the key?
- What is "Notify service user" referring to here -- their name? 

Is there a way we might clarify that even more for people?

Reply:
"Test" is the name of they key. Can change to something else if you want. Should be understood because it's under `api_key.name`.

"Notify service user" is the name of the internal Notify user, it will not change. We use it to associate a job with a user, when it's sent from the API.